src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+    
+body {
+  margin: 0;
+}
+  
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-left: 16px;
+  padding-bottom: 16px;
+}
+    
+.card__description {
+  font-family: "Roboto", sans-serif;
```

Comment:
You do not need to specify this style here, because you set it in `html` selector. Check other similar cases.